src/style.css

```diff
@@ -1 +1,71 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: bold;
+}
```

Comment:
There are two `@font-face` declarations for the 'Avenir' font using the same font file `fonts/Avenir-Book.ttf`. The second declaration adds a `font-weight: bold;`. If the same file is used for different font weights, it might not render correctly. Consider using different font files for different weights or ensure the font file supports multiple weights.